pkg/controllers/termination/suite_test.go

```diff
@@ -76,8 +75,10 @@ var _ = Describe("Termination", func() {
 					v1alpha2.ProvisionerNamespaceLabelKey: "default",
 				},
 			})
-			ExpectCreatedWithStatus(env.Client, node)
+			ExpectCreated(env.Client, node)
 			Expect(env.Client.Delete(ctx, node)).To(Succeed())
+			node = ExpectNodeExists(env.Client, node.Name)
```

Comment:
As a side note not for this PR, but I think we should add a check here to see if it has a `deletionTimestamp` set, for good measure.